pkgs/stdenv/custom/default.nix

```diff
@@ -2,7 +2,7 @@
 , localSystem, crossSystem, config, overlays
 }:
 
-assert crossSystem == null;
+assert crossSystem == localSystem;
```

Comment:
We're getting closer to changing these back to `buildPlatform` and `hostPlatform` :). Good discovery!